src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -148,32 +146,29 @@ public boolean link(ProcessPaths paths, ProjectConfiguration projectConfiguratio
         linkBuilder.command().addAll(getTargetSpecificLinkFlags());
         linkBuilder.redirectErrorStream(true);
         Process compileProcess = linkBuilder.start();
-        InputStream inputStream = compileProcess.getInputStream();
         int result = compileProcess.waitFor();
         if (result != 0 ) {
             System.err.println("Linking failed. Details from linking below:");
-            printFromInputStream(inputStream);
+            printFromInputStream(compileProcess.getInputStream());
```

Comment:
I'm ok with this change If it works. But I'm not sure it is possible to call getInputStream() after waitFor() has been called and still have the same inputStream?

Reply:
I'm not aware (and docs do no mention) of such a limitation.  Can revert if it does not work.